### ISSUE TITLE
Handling of unreleased version tags

### ISSUE THREAD

**Issue by jjlawren** [home-assistant/hass-release#26]
The PRs tagged with 0.101.4 (which was never released) were first made available in 0.102. However, these PRs were not mentioned in the 0.102 release notes. Perhaps unpublished versions should be cleaned up before/during the release procedure.

**Comment by balloob** [comment]
Right. I should have deleted the milestone instead of closing it, so that the script should have picked it up.
